packages/nimble-components/src/wafer-map/tests/computations.spec.ts

```diff
@@ -16,12 +16,17 @@ describe('Wafermap Computations module', () => {
         beforeEach(() => {
             const waferMock: Pick<
             WaferMap,
-            'dies' | 'originLocation' | 'canvasWidth' | 'canvasHeight'
+            | 'dies'
+            | 'originLocation'
+            | 'canvasWidth'
+            | 'canvasHeight'
+            | 'validity'
```

Comment:
Could you make a helper for creating the wafer mock for this file?

Edit: Actually looks like we should try and clean-up some of the other modules as well that create WaferMap Mocks. I don't think we should necessarily create a giant complex mock shared across all the tests but maybe one per test file. Or maybe a shared mock would still be cleaner.

Either way that shouldn't grow the scope of this PR. So can you create a tech debt task for the clean-up of WaferMap mocks and address it in a follow-up PR?

Reply:
sure, I made it here [Technical Debt 2520619](https://dev.azure.com/ni/DevCentral/_workitems/edit/2520619): Clean up existing nimble tests